scss/inc/_base.scss

```diff
@@ -420,12 +497,20 @@ body {
     // added to make _u_ in ck QTI compatible
     .txt-underline {
         text-decoration: underline;
-        font-size : inherit;
+        font-size: inherit;
+    }
+    .writing-mode-vertical-rl .txt-underline {
+        text-underline-position: right;
+    }
+    .writing-mode-vertical-rl[data-useragent-browser='safari'] .txt-underline {
+        // for Safari < 18.2 which doesn't support "text-underline-position: right"
+        text-decoration: none;
+        border-right: 1px solid currentColor;
```

Comment:
I'm not sure if `[data-useragent-browser='safari']` is really needed, we can always use `border-right: 1px solid currentColor;` instead of underline, in all browsers.

Just when testing I found that in Chrome, on some zooms/resolutions, border seems to be 1px farther to the right than underline, and so it overlaps ruby tags making them less readable. Safari seems fine.

Reply:
I don't have a preference. The extra attribute and coupling is a bit annoying, and it could be years before Safari 18.2 is the minimum supported browser, so if you want to change it for the `border-right` for all browsers it makes sense.

Also noticed that the `text-underline-position: right` disappears when the word is selected in Chrome, but the `border-right` does not.

Reply:
Interesting thing with selection!
It's difficult to choose, but I guess I'll leave `underline` because of ruby. It's just 1px, but they are already hard to read...